### PR TITLE
Feature/arm support

### PR DESCRIPTION
**Pull request by ahma**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

**Review by tarokkk — APPROVED** [review]

**Review by bonifaido — APPROVED** [review]

File: fluentd-image/v1.11/Dockerfile [hunk 1]
```diff
@@ -64,8 +64,8 @@ RUN apk update \
          fluent-plugin-redis \
  && gem specific_install -l https://github.com/cloudfoundry/fluent-plugin-syslog_rfc5424.git \
  && gem specific_install -l https://github.com/banzaicloud/fluent-plugin-gcs.git \
- && gem specific_install -l https://github.com/SumoLogic/sumologic-kubernetes-collection.git -d fluent-plugin-kubernetes-metadata-filter \
- && gem specific_install -l https://github.com/SumoLogic/sumologic-kubernetes-collection.git -d fluent-plugin-enhance-k8s-metadata \
+ && gem specific_install -l https://github.com/SumoLogic/sumologic-kubernetes-fluentd.git -d fluent-plugin-kubernetes-metadata-filter \
+ && gem specific_install -l https://github.com/SumoLogic/sumologic-kubernetes-fluentd.git -d fluent-plugin-enhance-k8s-metadata \
```
Comment (pepov) [inline fluentd-image/v1.11/Dockerfile:68]: is this change intentional?

**Review by pbalogh-sa — APPROVED** [review]

File: fluentd-image/v1.11/Dockerfile (hunk 1 above)
Comment (ahma) [reply fluentd-image/v1.11/Dockerfile:68]: yah, this plugins moved a new repo
